timescaledb/how-to-guides/hypertables/create.md

```diff
@@ -33,6 +33,17 @@ explicitly time-based, as long as it can increment. For example, a
 monotonically increasing ID works.
 </highlight>
 
+<highlight type="note"> 
+If your table has a `PRIMARY KEY` or `UNIQUE` constraint, that key or constraint
+must include all columns that are used to partition your hypertable. All
+TimescaleDB hypertables are partitioned by a time column, so the key or
+constraint must include the time column.
+
+It may additionally include columns that aren't partitioning columns. For more
+information, see [the troubleshooting
+section](https://docs.timescale.com/timescaledb/latest/how-to-guides/hypertables/create/#primary-keys-and-unique-constraints).
+</highlight>
```

Comment:
Not a big fan that we repeat this text multiple times (with minor changes). I'd prefer we add this to the section below and make a small note highlight that points to the larger section below so that we only have to edit one place later.

Maybe something like: "If you are trying to create a hypertable with a `PRIMARY KEY` or `UNIQUE` constraint and receiving an error, please see X for more information"